src/screens/UserInfoScreen/index.tsx

```diff
@@ -1,69 +1,39 @@
-import React, { FC, useEffect, useState } from 'react';
+import React, { FC } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { ReduxStoreType } from '../../redux/store';
 import { Image, Text, View } from 'react-native';
-import { TodoReducerState } from '../../redux/reducers/TodoReducer';
+import { TodoReducerState } from '../../redux/reducers/todoReducer';
 import { SignOutButton } from '../../components/SignOutButton';
 import { style } from './style';
 import { TodoItemType } from '../../models';
-import auth from '@react-native-firebase/auth';
-import { GoogleSignin } from '@react-native-google-signin/google-signin';
-import { signOutAction } from '../../redux/actions/todoActions/signOutAction';
-import { firebase } from '@react-native-firebase/database';
-import { SignInPayload } from '../../redux/actions/todoActions/signInAction';
-import { createReferenceHelper } from '../../helpers/createReferenceHelper';
+import { signOutSagaAction } from '../../redux/actions/authSagaActions/signOutSagaAction';
+import { userSelector } from '../../redux/selectors/userSelector';
+import { todoItemsSelector } from '../../redux/selectors/todoItemsSelector';
 
 export const UserInfoScreen: FC = () => {
-  const [data, setData] = useState<TodoItemType[]>([]);
   const dispatch = useDispatch();
 
-  const path = (userToken: SignInPayload['userToken']) => {
-    return `Users/${userToken}/Todo`;
-  };
-
   const userInfo = useSelector<ReduxStoreType, TodoReducerState['user']>(
```

Comment:
I told to you how to add types for useSelector, please check typescript section in redux docs